src/cartesian/Line.tsx

```diff
@@ -211,12 +211,18 @@ export class Line extends PureComponent<Props, State> {
   getStrokeDasharray = (length: number, totalLength: number, lines: number[]) => {
     const lineLength = lines.reduce((pre, next) => pre + next);
 
+    // if lineLength is 0 return the default when no strokeDasharray is provided
+    if (!lineLength) {
+      return `${length}px ${totalLength - length}px`;
```

Comment:
See lines below for where this comes from and why I returned this specifically 
https://github.com/recharts/recharts/pull/3904/files#diff-acabe8d166eb869ed5f168003bedf80f5bad4827ce727453b57733b3c0cf1974R439-R444

Reply:
This works, but it would be nicer to encapsulate in a helper function to communicate that both cases should be the same, right? 

```
function generateSimpleLine(totallength, length){
    return `${length}px ${totalLength - length}px`
}
```

Reply:
Sure, easy change